build.gradle

```diff
@@ -18,6 +18,7 @@ targetCompatibility = 1.6
 repositories {
     maven { url "http://oss.sonatype.org/content/repositories/snapshots" }
     maven { url "http://repo.maven.apache.org/maven2" }
+    mavenLocal();
```

Comment:
This is highlighting to me that our repository list is unusual, I think it should be:
```gradle
repositories {
    mavenLocal()
    mavenCentral()
}
```
If we're going to use the local repo we should check it in preference to central. I also don't think we need the snapshot repo url here as we should not normally build from snapshots and the uploads for snapshots are handled by a different repository element in the `mavenDeployer`.